service/src/test/java/bio/terra/workspace/service/resource/referenced/ReferenceResourceServiceTest.java

```diff
@@ -350,6 +353,9 @@ void testGcsBucketReference() {
             resultReferenceResource.castToGcsBucketResource();
         assertThat(resource, equalTo(resultResource));
 
+        referenceResourceService.validateReference(
```

Comment:
do you not need to check the return value of these?

Reply:
+1 here. I think you either need to check the return value or change `validateReference` to throw an exception if it doesn't exist instead.

Reply:
done